cylc/flow/main_loop/__init__.py

```diff
@@ -46,7 +46,7 @@
 
   .. code-block:: console
 
-     $ # run a workflow using the "health check" and "auto restart" plugins:
+     # run a workflow using the "health check" and "auto restart" plugins:
```

Comment:
This was correct before. Console lines should start with a PS1/2 otherwise they are considered command output.

Take a look at this example with the GH lexer to see the issue:

**console:**

```console
$ echo foo
foo
$ echo '# bar'
# bar
```

**bash:**

```bash
$ echo foo
foo
$ echo '# bar'
# bar
```

With pygments the result is worse.

Reply:
```suggestion
     $ # run a workflow using the "health check" and "auto restart" plugins:
```